config/admin-functions/create-machine-cert.sh

```diff
@@ -117,7 +117,7 @@ mkdir -p "${VX_CONFIG_ROOT}"
 
 # TODO: Use the machine's TPM private key (generated by the generate-key.sh script) instead of a
 # new key
-machine_private_key_password="$(generate_password_with_at_least_256_bits_of_entropy)"
+machine_private_key_password="$(generate_password_with_256_bits_of_entropy)"
```

Comment:
Missed this in https://github.com/votingworks/vxsuite-complete-system/pull/283/commits/d91692635c36ee9511d77e16eecde323fc5f1fba - I'm too used to TypeScript catching all my mistakes 😅